packages/fxa-auth-server/lib/senders/emails/partials/button/index.scss

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+@use '../../global.scss';
+
+.primary-button {
+  height: 56px !important;
+  width: 310px !important;
+
+  a {
+    @extend .font-sans;
+    @extend .text-lg;
+    background: global.$blue-400 !important;
+    width: 310px !important;
+    color: global.$white;
+    padding: 14px 0 !important;
```

Comment:
Ohh I missed this in my review I just submitted, this value should either be `12px` or `16px`.

I think our rule of thumb is becoming if you're setting `padding` or `margin`, use the variables provided. Though, I'll run this one by a designer and it can be fixed in the `cadReminder` ticket because I'm not sure which would be preferred.